KonyVisualizer.gitignore

```diff
@@ -0,0 +1,154 @@
+# Below list few files not added to gitignore list to support Headless/CI build
+
+# Auto-generated javascript. It's important to note that jssrc at the project root
+# can be ignored, but resources/customlibs/jsSrc must NOT be ignored because it stores
+# FFI binding code which can only be generated during development.
+/jssrc/*
+!/resources/customlibs/jsSrc
+
+# Starting with Visualizer 8.0 the Ant Contrib Jar used for the build process will
+# be regenerated when the build is fired. If you're using Visualizer 7.x then comment out
+# this line. Vis 7.x will not regenerate this jar and ignoring it will break your project.
+ant-contrib-0.6.jar
+
+# Binaries Ex:*.apk,*.KAR,
+binaries/
+
+# Other Visualizer auto-generated files.
+/.webmeta
```

Comment:
.webmeta and .meta can't be ignored as the metadata has ids connecting browser widgets to corresponding local html files. We don't checkin then the browser widget-to-localhtml associations will be lost.